packages/forms/test/form_array_spec.ts

```diff
@@ -64,6 +64,91 @@ describe('FormArray', () => {
 
       expect(a.controls).toEqual([c1, c2, c3]);
     });
+
+    it('should not emit event when pushing if emitEvent is false', fakeAsync(() => {
```

Comment:
I'd propose to format test title like this:
```suggestion
    it('should not emit events when calling `FormArray.push` with `emitEvent: false`', fakeAsync(() => {
```
So that it's easier to understand what the test does just by looking at the spec title.

(similar in other test cases too)